panel/src/panel/events.js

```diff
@@ -33,6 +33,15 @@ export default (panel) => {
 	emitter.on("keydown.cmd.shift.f", () => panel.search());
 	emitter.on("keydown.cmd./", () => panel.search());
 
+	/**
+	 * Custom copy to clipboard event
+	 * @since 5.0.0
+	 */
+	emitter.on("copyToClipboard", async (e) => {
+		navigator.clipboard.writeText(e);
+		panel.notification.success(panel.t("copied") + "!");
+	});
```

Comment:
`Discuss:` I'm wondering if this is the best as global event or if rather DropdownItem/DropdownContent should handle this? 

Also, shouldn't we use `$helper.clipboard.write()` then? Or adapt the helper method to this code?

Reply:
Woah, I completely forgot about the clipboard helpers 🙈 I just replaced that part. I'm also not entirely sure about the event. We could maybe provide some handy action shortcuts and copy could be one of them. But we also have to be careful that we don't make this all too complicated.

Reply:
Thinking about it some more, the global event might be good. Can then also be used by e.g. dialog success events. But how about simplifying the name to just `clipboard`?

Reply:
Maybe we could do `clipboard.write` and then there's still an open slot for `clipboard.read` later?

Reply:
I've already renamed it to clipboard.write. If you don't like it, we can still change it.